es-app/src/FileSorts.cpp

```diff
@@ -36,8 +36,9 @@ namespace FileSorts
 	//returns if file1 should come before file2
 	bool compareFileName(const FileData* file1, const FileData* file2)
 	{
-		std::string name1 = file1->getName();
-		std::string name2 = file2->getName();
+		// we compare the actual metadata name, as virtual files have the system appended which messes up the order
+		std::string name1 = file1->metadata.get("name");
+		std::string name2 = file2->metadata.get("name");
```

Comment:
LGTM. We were no longer comparing filenames a long time ago, though. We are comparing File names :)

But yes, I can certainly change it.